src/server/pkg/backoff/exponential.go

```diff
@@ -114,8 +127,24 @@ func NewInfiniteBackOff() *ExponentialBackOff {
 	} else if b.RandomizationFactor > 1 {
 		b.RandomizationFactor = 1
 	}
-	b.Reset()
-	return b
+	return b.withCanonicalRandomizationFactor().withReset()
+}
+
+// New10sBackoff returns a backoff that's slightly more aggressive than
+// NewExponentialBackOff. The Max Elapsed time for this backoff is 10s, and the
+// initial backoff is 100ms (instead of 500). Therefore this will retry at most
+// 11 times and then fail (depending on RPC timeout), and may be more useful
+// for interactive RPCs than the default timeout of 60s.
+func New10sBackoff() *ExponentialBackOff {
```

Comment:
Seems kinda weird that we have a function specifically for 10s.  Would it be hard to simply use `NewExponentialBackoff`, and then manually set the `MaxElapsedTime` before using it?